lib/src/buttons/loading_button.dart

```diff
@@ -113,6 +117,15 @@ class _MUILoadingButtonState extends State<MUILoadingButton> {
         duration: Duration(milliseconds: widget.animationDuration),
         decoration: BoxDecoration(
           borderRadius: BorderRadius.circular(widget.borderRadius),
+          boxShadow: widget.shadow
+              ? [
+                  const BoxShadow(
+                    color: Color.fromRGBO(72, 76, 82, 0.16),
+                    offset: Offset(0, 12),
+                    blurRadius: 16.0,
+                  )
+                ]
+              : [],
```

Comment:
Instead of this condition just `widget.boxShadow`.

Reply:
done